campaigns/inferno_deal_with_the_devil.tex

```diff
@@ -187,18 +191,14 @@ \subsection*{\MakeUppercase{Additional Rules}}
     else. They also make use of Stables 1 MP bonus.
   \item Enemy Charging Heros always move amongst each to not ``block'' each other's paths (thus slowing each
     other down). The Enemy Charging Hero that can get closest to Kleesive will move before any others.
-  \item Castle and Rampart already have all Mines controlled in their territory (non-Inferno Map Tiles). Place enemy
-    Cubes on all of their Mines.
   \item Your Walls, Gates, and Arrow Tower in Kleesive are completely rebuilt after each battle.
   \item Enemy \svg{unit_flying} Units will fly over Walls and Gates to attack your Units whenever possible.
   \item Enemy \svg{unit_ranged} Units will always attack the Arrow Tower once there are no other \svg{unit_ranged} Units to attack.
   \item Enemy Units will always attack your Units if possible. Only when that is not possible, then the Gate,
     and then Walls will be attacked.
 \end{itemize}
 
-\vspace*{\fill}
-\includegraphics[width=\linewidth, keepaspectratio]{\art/cerberi.png}
-\vspace*{\fill}
+\includegraphics[width=\columnwidth, keepaspectratio]{\art/cerberi.png}
```

Comment:
```suggestion
\vspace{2em}
\includegraphics[width=\columnwidth, keepaspectratio]{\art/cerberi.png}
```
Those dogs are a little too close to the text now. Try moving them lower, like this.

Reply:
Or thy the same trick you commented out with demon. Both should work fine.